FrEIA/modules/base.py

```diff
@@ -0,0 +1,38 @@
+from typing import Tuple, Iterable, Union
+
+import torch.nn as nn
+from torch import Tensor, is_tensor
+
+
+class InvertibleOperator(nn.Module):
```

Comment:
`InvertibleModule`?

Reply:
does it cover the contract of a `nn.Module`? If so, then **yes** change the name accordingly.

Reply:
Yes, it does